src/content/docs/tutorial-kubernetes-learn/tutorial-k8s-intro.mdx

```diff
@@ -0,0 +1,148 @@
+---
+title: Understand your kubernetes system with New Relic
+---
+
+import tutorialNavigator from 'images/kubernetes_screenshot-full_navigator-main.webp'
+
+import tutorialInstall from 'images/kubernetes_screenshot-full_guidedInstall.webp'
+
+import k8sNavigatorAnimation from 'images/kubernetes_screenshot-full_navigator-animated.gif'
+
+import apmApmCompleteSummaryPage from 'images/apm_screenshot-full_apm-complete-summary-page-.webp'
+
+import infraComplete from 'images/infrastructure_screenshot-full_infra-main.webp'
+
+
+A Kubernetes system is inherently complex and Kubernetes as a whole has a reputation as intricate and challenging to grasp. Understanding Kubernetes as a concept is a complex task by itself and wanting to understand an indivdual Kubernetes system only adds more complextiy. How do you troubleshoot ephemeral containers that spin up and down before you can even access them? How do you understand the health of your system as a whole if you have hundreds of containers orchestrated at a time?
+
+In this tutorial series, we will explore how you can monitor your Kubernetes system with New Relic and how it can streamline your understanding of the technology.
+
+          <img
+            title="New Relic navigator"
+            alt="A screenshot showing the New Relic navigator for Kubernetes. It shows indivudual pods, their health, and operational groups."
+            src={tutorialNavigator}
+          />
+
+<p></p>
+<p></p>
+
+##  Get started:
+
+To begin, you need to set up New Relic with your Kubernetes system. The steps below guide you though the process:
+
+<Steps>
+    <Step>
+
+### Install New Relic agent: 
+
+
+The New Relic Kubernetes integration gives you full observability into the health and performance of your environment by leveraging the New Relic infrastructure agent. This agent collects telemetry data from your cluster using several New Relic integrations such as the Kubernetes events integration, the Prometheus Agent, and the New Relic Logs Kubernetes plugin.
```

Comment:
What if I don't know what all of these nouns mean? :) Can you de-jargon this a bit?